app/views/clients/_form.html.haml

```diff
@@ -1,4 +1,6 @@
-= form_for(@client, html: { class: 'form-horizontal' }) do |f|
+= semantic_form_for(@client, html: { class: 'form-horizontal' }) do |f|
+  = f.semantic_errors(*f.object.errors.keys)
```

Comment:
seems like you should also delete all the hackery i put in there to do this exact tame thing?  doesn't this line replace all of https://github.com/apurvis/rader/blob/master/app/views/clients/new.html.haml ?  (don't trust me though; i'm just guessing)